### ISSUE TITLE
Chiaki on mac won't run

### ISSUE THREAD

**Issue by Snowlax14** [thestr4ng3r/chiaki#15]
When I try to start Chiaki it gives me a (OSStatus error -67033.) 
Apple says that the issue is: 
This error is returned when the system is attempting to sign unsigned code ad-hoc and couldn’t write to the signature database because of a permission problem.

**Comment by thestr4ng3r** [comment]
Have you dragged the app to your Applications directory? Normally macOS refuses to run the app by default and you have to go through Settings/Security/... to explicitly allow it, but that error is new to me.

**Comment by Snowlax14** [comment]
Yes I moved it and allowed it. Also tried resigning it but still gives me an error.

**Comment by thestr4ng3r** [comment]
Can you please try this? It's a signed build. Since you already said you tried to sign it, I don't have much hopes, but doesn't hurt to try. [chiaki-signed.tar.gz](https://github.com/thestr4ng3r/chiaki/files/3529782/chiaki-signed.tar.gz)

**Comment by Snowlax14** [comment]
No it still gives me same error :(

**Comment by thestr4ng3r** [comment]
It's weird, I can't find anything about this error. For me, everything works fine on Mojave. What macOS version do you have?

**Comment by Snowlax14** [comment]
I have Mojave too (Version 10.14.5)

**Comment by kercdbh** [comment]
Same here, cannot open the app; here is my info:
`$ sw_vers
ProductName:	Mac OS X
ProductVersion:	10.11.6
BuildVersion:	15G31
$ open -a Chiaki
LSOpenURLsWithRole() failed for the application /Applications/Chiaki.app with error -10810.`

**Comment by derusc** [comment]
I have the same issue, executing directly from the content I have a clearer message:
```
$ ./Applications/Chiaki.app/Contents/MacOS/chiaki 
Sorry, "Chiaki" cannot be run on this version of macOS. Qt requires macOS 10.12.0 or later, you have macOS 10.11.6.
```

**Comment by thestr4ng3r** [comment]
> I have the same issue, executing directly from the content I have a clearer message:
> 
> ```
> $ ./Applications/Chiaki.app/Contents/MacOS/chiaki 
> Sorry, "Chiaki" cannot be run on this version of macOS. Qt requires macOS 10.12.0 or later, you have macOS 10.11.6.
> ```

Well that one is pretty obvious. El Capitan is already quite old, it doesn't make too much sense to support this in the official builds, sorry.

**Comment by derusc** [comment]
Thanks Florian,
I just wanted to make it clear for other people that may get the same issue.

Thanks for the great work!
